apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/ChatInfoView.kt

```diff
@@ -1308,6 +1343,77 @@ fun queueInfoText(info: Pair<RcvMsgInfo?, ServerQueueInfo>): String {
   return generalGetString(MR.strings.message_queue_info_server_info).format(json.encodeToString(qInfo), msgInfo)
 }
 
+fun setChatTTLAlert(
+  m: ChatModel,
```

Comment:
this pattern transferring chatModel by param is gone couple of years ago. Just declare it inside a function if you need it named `m` like this:
val m = chatModel